test/ex_member_page_test.rb

```diff
@@ -1,20 +1,27 @@
 # frozen_string_literal: true
 require_relative './test_helper'
 require_relative '../lib/ex_member_page.rb'
+require_relative '../lib/decorators/formatted_dates'
+require_relative '../lib/electoral_history'
```

Comment:
you should really only be requiring things that the test itself is directly using. If you actually _do_ need to require these here, then something seems off.